user-image/connectors/neuro.bash

```diff
@@ -15,3 +15,6 @@ git checkout b4dd6d860d7c27a846a5445b0df2824429084b44
 python setup.py install
 cd .. && rm -rf pycortex
 rm -rf ~/.config/pycortex
+
+# Install the package nistats (currently not on pypi yet)
+(cd ~ && git clone https://github.com/nistats/nistats.git && cd nistats && python setup.py install && cd ~ && rm nistats -rf)
```

Comment:
@eickenberg Could you remove the ( ) and split the command up over multiple lines for clarity/to match rest of formatting? Thank you!

Reply:
can totally do that. So I can break the && chain then?

Reply:
Yes, please!

Reply:
OK, made that change. Does this look better?